src/Runner.Worker/JobRunner.cs

```diff
@@ -49,6 +49,9 @@ public async Task<TaskResult> RunAsync(AgentJobRequestMessage message, Cancellat
                 VssCredentials jobServerCredential = VssUtil.GetVssCredential(systemConnection);
                 await runServer.ConnectAsync(systemConnection.Url, jobServerCredential);
                 server = runServer;
+
+                _jobServerQueue = HostContext.GetService<IJobServerQueue>();
+                _jobServerQueue.Start(message, enableActionsUpload: false);
```

Comment:
the name `enableActionsUpload` is kind of confusing. :D 
my first read is something about uploading an Action as an artifact or something.

Reply:
😄  Do you have a suggestion?  The intention is to skip any http interactions with Actions service with this bool flag.

Reply:
Does `enableTimelineRecordsUpload` make sense?

Reply:
It's not only timeline records upload though, we also create attachments and upload log content.  

Maybe `enableInteractionsWithActions`? 🤷🏼

Reply:
`resultServiceOnly = true`? and have it default to `false` so you don't have to change the other caller. 😄

Reply:
Renamed the variable.